frame/dapps-staking/src/lib.rs

```diff
@@ -60,9 +60,9 @@ impl Default for Forcing {
 #[derive(PartialEq, Eq, Clone, Default, Encode, Decode, RuntimeDebug, TypeInfo)]
 pub struct EraRewardAndStake<Balance> {
     /// Total amount of rewards for an era
-    rewards: Balance,
+    pub rewards: Balance,
```

Comment:
For this change you need version bump

Reply:
I will update pallet rev after rebasing to the un-bonding PR #509